modules/admin_manual/examples/configuration/oc_permit.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+ocpath='/var/www/owncloud'
+ocstorage='/mnt/owncloud'
```

Comment:
If their storage is in "/mnt/..." then this line is correct.

otherwise should be:
ocstorage='/var/www/owncloud'

Reply:
Fixed with a note that they might need to adjust the paths in the script